feature-libs/product-configurator/rulebased/components/attribute/types/numeric-input-field/configurator-attribute-numeric-input-field.component.ts

```diff
@@ -49,19 +55,42 @@ export class ConfiguratorAttributeNumericInputFieldComponent
     //locales are available as 'languages' in the commerce backend
     this.locale = this.getInstalledLocale(this.language);
 
+    let numDecimalPlaces = this.attribute.numDecimalPlaces;
+    let numTotalLength = this.attribute.numTotalLength;
+    let negativeAllowed = this.attribute.negativeAllowed;
+
+    if (
+      numDecimalPlaces === undefined ||
+      numTotalLength === undefined ||
+      negativeAllowed === undefined
+    ) {
```

Comment:
Is it possible to write like this ` if (numDecimalPlaces || numTotalLength  || negativeAllowed)` ?
It should be the same

Reply:
No, not the same. 
Suppose numDecimalPlaces is 0. Then it is not undefined, but `if (numDecimalPlaces)` evaluates to true